src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -0,0 +1,136 @@
+<form
+  role="form"
+  ng-submit="stepTimeField.createIndexPattern()"
+  class="kuiPanel"
+>
+  <div class="kuiPanelHeader">
+    <div class="kuiPanelHeaderSection">
+      <h2 class="kuiPanelHeader__title">
+        Step 2 of 2: Optional settings
+      </h2>
+    </div>
+  </div>
+
+  <div class="kuiPanelBody">
+    <p class="kuiText kuiSubduedText kuiVerticalRhythm">
+      You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some optional settings before we create it.
+      <small>
+        <a
+          class="kuiLink"
+          ng-click="stepTimeField.toggleAdvancedOptions()"
+        >
```

Comment:
I actually really like the second suggestion too. I went ahead and added it.

Do you think we should toggle the text between `Show` and `Hide`?

<img width="1152" alt="screen shot 2017-08-14 at 11 01 27 am" src="https://user-images.githubusercontent.com/56682/29277551-f4896db6-80df-11e7-9fb1-f340ff636874.png">
<img width="1150" alt="screen shot 2017-08-14 at 11 01 31 am" src="https://user-images.githubusercontent.com/56682/29277550-f48397d8-80df-11e7-9fbc-592ff0e76a03.png">